packages/react-component-ref/src/RefFindNode.tsx

```diff
@@ -32,13 +32,17 @@ export default class RefFindNode extends React.Component<RefFindNodeProps> {
     handleRef(this.props.innerRef, this.prevNode)
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps: RefFindNodeProps) {
```

Comment:
think that it might be beneficial to implement this component with Hooks, for couple of reasons
- in that case we will have explicit update dependencies specified in `useEffect`
- explicitness on `unsubscribe` actions for effects
- this component is fairly small in terms of amount of code